lib/services/webpack/webpack-compiler-service.ts

```diff
@@ -155,6 +156,15 @@ export class WebpackCompilerService extends EventEmitter implements IWebpackComp
 		}
 	}
 
+	private async shouldUsePreserveSymlinksOption(projectData: IProjectData): Promise<boolean> {
```

Comment:
Do you need to pass `projectData` here? 😸

Reply:
Nice catch.